ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3917,3 +3917,38 @@ def test_tensorflow_Zeta(
         x=x[0],
         q=x[1],
     )
+
+
+# Imag
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Imag",
+    dtype_and_xs=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("complex"),
+    ),
+    Tout=st.booleans(),
```

Comment:
I guess that is wrong. You should sample from `float32` and `float64` instead

Reply:
This shouldnt be wrong as tf as far as ik takes only complex inputs 
![image](https://github.com/unifyai/ivy/assets/41777835/8e759045-7dba-41d2-b22a-b548b18c1667)

Reply:
I meant for `Tout`, as per our conversation privately. You should remove reference of `Tout` completely from here.

Reply:
I have used the test code for tf.raw_ops.Angle as it is because 1.It works and checks all the boxes included in the deep dive docs and 2. They are more or less the same function (Both take a complex input and one returns the Imaginary component while the other returns the angle from the real axis)